pkg/kudoctl/packages/writer/writer_tar.go

```diff
@@ -112,5 +112,9 @@ func TgzDir(fs afero.Fs, path string, w io.Writer) (err error) {
 
 // packageVersionedName provides the version name of a package provided a set of Files.  Ex. "zookeeper-0.1.0"
 func packageVersionedName(pkg *packages.Files) string {
-	return fmt.Sprintf("%v-%v", pkg.Operator.Name, pkg.Operator.Version)
+	if pkg.Operator.AppVersion == "" {
+		return fmt.Sprintf("%v-%v", pkg.Operator.Name, pkg.Operator.OperatorVersion)
```

Comment:
this line L116 has a different return than L119.  It thought I read a request by Alex to change `-` to `_`.  I will investigate and change accordingly.

Reply:
I was wrong here... I just needed to review more closely... this looks good.